src/config/theme.js

```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
   ...bodhiTheme.overrides,
   MuiButton: {
     root: {
+      fontSize: xSmallText,
+      lineHeight: lineHeight24,
+      padding: `4px ${px(space2X)}`,
+      fontWeight: fontWeightBold,
       borderRadius: px(space7X),
       textTransform: 'none',
+      minWidth: 'auto',
+      minHeight: 'auto',
       [bodhiTheme.breakpoints.down('xs')]: {
         padding: px(spaceX),
         minWidth: 64,
         minHeight: 32,
         fontSize: xxSmallText,
       },
     },
-    raisedPrimary: {
+    contained: {
       backgroundColor: primaryBrandColor,
       color: white,
-      boxShadow: '0px 0px 2px rgba(0, 0, 0, .5)',
+      boxShadow: 'none',
       '&:hover': {
-        backgroundColor: primaryBrandColor,
-        boxShadow: '0px 0px 10px rgba(0, 0, 0, .25)',
+        backgroundColor: primaryBrandColor80,
+        boxShadow: 'none',
+      },
+      '&:active': {
+        backgroundColor: primaryBrandColor60,
+        boxShadow: 'none',
       },
       '&:disabled': {
-        backgroundColor: primaryBrandColor,
-        color: white,
-        opacity: 0.6,
+        backgroundColor: primaryBrandColor30,
+        color: `${white}!important`,
+        boxShadow: 'none',
+      },
+    },
+    outlined: {
+      color: primaryBrandColor,
+      border: `1px solid ${primaryBrandColor}`,
+      boxShadow: 'none',
+      '&:hover': {
+        backgroundColor: primaryBrandColor30,
+        boxShadow: 'none',
+      },
+      '&:active': {
+        backgroundColor: primaryBrandColor60,
+        boxShadow: `1px 1px 4px ${primaryBrandColor60}`,
+      },
+      '&:disabled': {
+        color: primaryBrandColor30,
+        border: `1px solid ${primaryBrandColor30}!important`,
+        boxShadow: 'none',
+      },
+    },
+    text: {
+      color: primaryBrandColor,
+      boxShadow: 'none',
+      '&:hover': {
+        backgroundColor: primaryBrandColor30,
+        boxShadow: 'none',
+      },
+      '&:active': {
+        backgroundColor: primaryBrandColor60,
+        boxShadow: `1px 1px 4px ${primaryBrandColor60}`,
+      },
+      '&:disabled': {
+        color: primaryBrandColor30,
+        boxShadow: 'none',
       },
     },
     sizeLarge: {
-      fontSize: mediumText,
+      fontSize: smallText,
+      lineHeight: lineHeight24,
+      padding: `12px ${px(space2X)}`,
       fontWeight: fontWeightBold,
-      minHeight: 56,
+      minWidth: 'auto',
+      minHeight: 'auto',
```

Comment:
do we need to set min width/height?

Reply:
I set these is to override the default values.